environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentCreationService.java

```diff
@@ -142,13 +142,13 @@ private Environment initializeEnvironment(EnvironmentCreationDto creationDto) {
     private void initializeEnvironmentTunnel(Environment environment) {
         Tunnel tunnel = environment.getExperimentalFeaturesJson().getTunnel();
         boolean ccmV2Enabled = entitlementService.ccmV2Enabled(environment.getAccountId());
-        if (Tunnel.CCMV2 == tunnel && !ccmV2Enabled) {
+        if ((Tunnel.CCMV2 == tunnel || Tunnel.CCMV2_JUMPGATE == tunnel) && !ccmV2Enabled) {
```

Comment:
useCcmV2OrJumpgate can be used here right ?

Reply:
right, if tunnel is not null (which I hope is not null)